onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -1134,54 +1136,133 @@ def generate_gpt2_init_decoder(
         [0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
     )
 
+    # Normalization Node is : SkipLayerNormalization
+    if logits_matmul_to_residual_add_path is None:
```

Comment:
Will think of a way to add a test for this in the next round of misc changes (have some more in the backlog). This is just to quickly unblock main branch.